code/datums/tutorial/xenomorph/xenomorph_basic.dm

```diff
@@ -0,0 +1,229 @@
+	addtimer(CALLBACK(src, PROC_REF(on_inform_health)), 10 SECONDS)
+
+/datum/tutorial/xenomorph/basic/proc/on_inform_health()
+	message_to_player("The green icon on the <b>right</b> of your screen and green bar next to your character represents your health.")
+	addtimer(CALLBACK(src, PROC_REF(on_give_plasma)), 10 SECONDS)
+
+/datum/tutorial/xenomorph/basic/proc/on_give_plasma()
+	message_to_player("You have been given <b>plasma</b>, a resource used for casting your abilities. This is represented by the blue icon at the <b>right</b> of your screen and the blue bar next to your character.")
+	xeno.plasma_max = 200
+	xeno.plasma_stored = 200
+	addtimer(CALLBACK(src, PROC_REF(on_damage_xenomorph)), 15 SECONDS)
+
+/datum/tutorial/xenomorph/basic/proc/on_damage_xenomorph()
+	xeno.apply_damage(350)
+	xeno.emote("hiss")
+	message_to_player("Oh no! You've been damaged. Notice your green health bars have decreased. Xenomorphs can recover their health by standing or resting on weeds.")
+	addtimer(CALLBACK(src, PROC_REF(request_player_plant_weed)), 10 SECONDS)
+
+/datum/tutorial/xenomorph/basic/proc/request_player_plant_weed()
+	update_objective("Plant a weed node using the new ability <b>Plant Weeds</b> you've just been given.")
+	give_action(xeno, /datum/action/xeno_action/onclick/plant_weeds)
+	message_to_player("Plant a weed node to spread weeds using your new ability at the top of the screen. Weeds heal xenomorphs and regenerate their plasma. They also slow humans, making them easier to fight.")
+	RegisterSignal(xeno, COMSIG_XENO_PLANT_RESIN_NODE, PROC_REF(on_plant_resinode))
+
+/datum/tutorial/xenomorph/basic/proc/on_plant_resinode()
+	SIGNAL_HANDLER
+	UnregisterSignal(xeno, COMSIG_XENO_PLANT_RESIN_NODE)
+	message_to_player("Well done. You can rest on the weeds to heal faster using the <b>Rest</b> ability or with the [retrieve_bind("rest")] key.")
+	message_to_player("We have increased your plasma reserves. Notice also your plasma will regenerate while you are on weeds.")
+	give_action(xeno, /datum/action/xeno_action/onclick/xeno_resting)
+	update_objective("Rest or wait until you are at least [WAITING_HEALTH_THRESHOLD] health.")
+	xeno.plasma_max = 500
+	RegisterSignal(xeno, COMSIG_XENO_ON_HEAL_WOUNDS, PROC_REF(on_xeno_gain_health))
+
+/datum/tutorial/xenomorph/basic/proc/on_xeno_gain_health()
+	SIGNAL_HANDLER
+	UnregisterSignal(xeno, COMSIG_XENO_ON_HEAL_WOUNDS)
+	message_to_player("Even on weeds. Healing is a slow process. This can be sped up using pheromones. Emit \"Recovery\" pheromones now using your new ability to speed up your healing.")
+	give_action(xeno, /datum/action/xeno_action/onclick/emit_pheromones)
+	update_objective("Emit recovery pheromones.")
+	RegisterSignal(xeno, COMSIG_XENO_START_EMIT_PHEROMONES, PROC_REF(on_xeno_emit_pheromone))
+
+/datum/tutorial/xenomorph/basic/proc/on_xeno_emit_pheromone(emitter, pheromone)
+	SIGNAL_HANDLER
+	if(!(pheromone == "recovery"))
```

Comment:
are there defines for these?

Reply:
There are not they're just strings.
https://github.com/cmss13-devs/cmss13/blob/2d26ea2db52a901fd363a76131d00233c94ed1dd/code/modules/mob/living/carbon/xenomorph/castes/caste_datum.dm#L56